examples/pipelines/asl_nistats_processing.py

```diff
@@ -0,0 +1,170 @@
+"""ASL processing with nistats
+"""
+
+import os
+import os.path as op
+import numpy as np
+import nibabel as nb
+import glob
+import pandas as pd
+import matplotlib.pyplot as plt
+import nilearn
+import scipy
+
+from collections import OrderedDict
+from nistats import experimental_paradigm, design_matrix, glm
+from nistats.glm import FirstLevelGLM
+from nistats.thresholding import map_threshold
+from nilearn.input_data import NiftiMasker
+from nilearn.plotting import plot_stat_map
+
+from pyhrf.retreat.glm_tools import make_design_matrix_asl
+from pypreprocess.reporting.glm_reporter import group_one_sample_t_test
+from pypreprocess.external.nistats.glm import FMRILinearModel
+
+from . import nistats_processing_tools as nt
+
+
+
+archives = op.join('/home', 'aina', 'Data', 'HEROES_preprocessed')
+mask_fn = op.join(archives, 'brain_mask_willard.nii')
+
+t_r = 2.5
+n_scans = 164
+frametimes = np.linspace(0, (n_scans - 1) * t_r, n_scans)
+
+drift_model = 'Cosine'
+hrf_model = 'Canonical'
+_duration = 10
+epoch_duration = _duration * t_r
+hfcut = 2 * 2 * epoch_duration
+
+subject_names = ["AC150013", "CD110147", "KP140463", "MP130368", "SC120530",
+                 "NS110383", "PB130006", "SB150062", "MB140004", "XM140202",
+                 "RG130377", "SH140342", "AM140315", "LZ140352", "HB140194"]
+subject_names = ["SH140342", "AM140315", "LZ140352", "HB140194"]
+
+first_levels = []
+mask_fn = op.join(archives, 'gm_mask_willard_asl.nii')
+mask_img = nb.load(mask_fn)
+masker = NiftiMasker(mask_img=mask_img).fit()
+affine = mask_img.get_affine()
+
+for subject in subject_names:
+    print subject
+    #mvt_file = glob.glob(op.join(archives, subject, 'Preprocessed', 'rp_sc_tag_ctl_vismot2.txt
+    anat_fn = glob.glob(op.join(archives, subject, 'Preprocessed', 'wranat_*.nii'))[0]
```

Comment:
avoid acronyms, people may not understand.
I suggest `anat_filename` or just `anat`